react-native/architecture/project-architecture.s.md

```diff
@@ -2,11 +2,44 @@
 
 ## Checks
 - Entire code except `index.ios.js` and `index.android.js` should be in `src`
-- The structure should be as follow
+- The structure should be as follow and follow the linked standards
   - `index.ios.js`
```

Comment:
On peut tenter ça pour plus de lisibilité ? https://stackoverflow.com/questions/19699059/representing-directory-file-structure-in-markdown-syntax

Reply:
Je peux le faire, si ok.